### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by Metodija-Mitovski**
Vlado please review this code

### CODE REVIEW

**Review by VladimirMachkovski — COMMENTED** [review]
OK i will check it

File: Colegues/Mete/Mete.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mete html</title>
+  </head>
+  <body>
+    <h1>Html content</h1>
```
Comment (VladimirMachkovski) [inline Colegues/Mete/Mete.html:10]: delete the capitol letter

Comment (Metodija-Mitovski) [reply Colegues/Mete/Mete.html:10]: ok i will delete as soon as possible

File: Colegues/Mete/Mete.html [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mete html</title>
```
Comment (Metodija-Mitovski) [inline Colegues/Mete/Mete.html:7]: ova treba da e mete page

File: Colegues/Mete/Mete.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+  </head>
+  <body>
+    <h1>Html content</h1>
+    <h1>Metodija</h1>
+  </body>
```
Comment (Metodija-Mitovski) [inline Colegues/Mete/Mete.html:12]: this tag is body

**Comment by VladimirMachkovski** [comment]
ok you can merged

**Pull request merged**